drizzlepac/devutils/comparison_tools/read_hla/1AAREADME

```diff
@@ -0,0 +1,16 @@
+Wed Jun 12 15:10:06 EDT 2019
+Functions to read HLA images and catalogs and apply the astrometric corrections
+from HSCv3.
+
+read_hla_image.py   Functions to read HLA image and apply WCS corrections
+read_hla_catalog.py Functions to read HLA catalog and apply WCS corrections
+omegaxyz.py         Functions to access WCS correction database and apply them
+
+See the test code at the bottom of these modules for examples of how to use them.
+
+hla_test.py         Test script that creates both FITS images with and without
+                    corrections and also HLA catalogs with and without corrections.
+
+dods9.csh           Shell script that displays the test images in ds9 and overlays
+                    the catalogs with (green) and without (red) the WCS corrections
+                    as a test.
```

Comment:
Update documentation here.

Reply:
Issue resolved in commit [5d67d60](https://github.com/spacetelescope/drizzlepac/pull/548/commits/5d67d60f9d5c245fa1ba9514ce38324b87b1af5d)